pkg/agent/const.go

```diff
@@ -73,6 +73,7 @@ const (
 // cloud-init (i.e. ARM customData) source file references
 const (
 	kubernetesNodeCustomDataYaml  = "linux/cloud-init/nodecustomdata.yml"
+	kubernetesCSEStartScript      = "linux/cloud-init/artifacts/cse_start.sh"
 	kubernetesCSECommandString    = "linux/cloud-init/artifacts/cse_cmd.sh"
```

Comment:
super nit: can we switch the order of L76 and L77? I'd like to see them in a logical order

Reply:
fixed